### PR TITLE
fix #33851 flyblog.cc

### LINKED ISSUE #33851: flyblog.cc
### Issue URL (Ads)
[http://flyblog.cc/blog/post/48915999](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=http%3A%2F%2Fflyblog.cc%2Fblog%2Fpost%2F48915999)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
Platform: | Ext
AdGuard version: | 3.0.14
Browser: | Chrome
Stealth mode options: | Hide your search queries, Send Do-Not-Track header, Remove X-Client-Data header from HTTP requests, Strip URLs from tracking parameters, Self-destructing third-party cookies (2880), Block WebRTC, Hide your Referrer from third-parties
Filters: | EasyList China,<br>EasyPrivacy,<br>AdGuard Turkish,<br>AdGuard French,<br>AdGuard Base,<br>Adblock Warning Removal List,<br>(Obsolete) Anti-Adblock Killer | Reek,<br>Fanboy's Enhanced Tracking List,<br>ChinaList+EasyList,<br>AdGuard Tracking Protection,<br>AdGuard German,<br>AdGuard Japanese,<br>AdGuard Spanish/Portuguese

### PR DESCRIPTION
**Pull request by zloyden**
https://github.com/AdguardTeam/AdguardFilters/issues/33851

### CODE REVIEW

File: EnglishFilter/sections/foreign.txt [hunk 1]
```diff
@@ -791,6 +791,11 @@ kliuki.bg###container > div[style="float:left;"] > div[style]
 !---------------------------------------------------------------------
 ! https://github.com/AdguardTeam/AdguardFilters/issues/33593
 kkphm.com,haohaiyo.cc,51kkp.club##a[href*="kkpmh.com/"]
+! https://github.com/AdguardTeam/AdguardFilters/issues/33851
+flyblog.cc##iframe[src*="ifoodie.tw/"]
```
Comment (BlazDT) [inline EnglishFilter/sections/foreign.txt]: Unsure about this one, looks like annoyance to me.

**Review by BlazDT — CHANGES_REQUESTED** [review]
Missed social left-over:
https://www.facebook.com/plugins/like.php?

**Comment by zloyden** [comment]
Removed social left-over.

**Review by BlazDT — APPROVED** [review]

File: EnglishFilter/sections/foreign.txt (hunk 1 above)
Comment (Alex-302) [reply EnglishFilter/sections/foreign.txt]: or not even annoyance

Comment (zloyden) [reply EnglishFilter/sections/foreign.txt]: Well, I deleted this rule.

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**